docs/core/install/linux-scripted-manual.md

```diff
@@ -77,6 +77,12 @@ For .NET apps that use the *System.Drawing.Common* assembly, you'll also need th
 
 The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the **SDK** and **Runtime**. You can download the script from <https://dot.net/v1/dotnet-install.sh>.
 
+The script currently requires bash. On some distros this needs to be installed. On Alpine, for example, you would install bash with:
+
+```bash
+apk add bash
+```
+
```

Comment:
For simplicity, I would just make this generic.

```suggestion
>![IMPORTANT]
>Bash is required to run the script.
```

Reply:
Note that in the distro specific topics we helpfully share the command to use, eg https://docs.microsoft.com/en-us/dotnet/core/install/linux-alpine#dependencies

But I think we can assume folks know (or can figure out) how to install it.